templates/dashboard.template.html

```diff
@@ -13,11 +13,7 @@
 
 <interstellar-basic-client-balance></interstellar-basic-client-balance>
 
-<div class="so-back">
```

Comment:
Unfortunately this `so-back` is what gives padding, and `so-chunk` gives the column its max width. We want to conditionally style this portion of the DOM when the transaction is pending, but this template doesn't have the data we need....